pkg/util/k8s/node.go

```diff
@@ -57,3 +58,26 @@ func GetNodeAddrs(node *v1.Node) (*ip.DualStackIPs, error) {
 	}
 	return nodeAddrs, nil
 }
+
+// GetNodeAddressFromAnnotations gets available IPs from the Node Annotation, the annotations are set by Antrea, includes
```

Comment:
Node Annotation, the -> Node Annotation. The

includes -> including

Reply:
Done.